{"timestamp":1729172829415,"message":"Unable to get edition with id 1","log.level":"ERROR","logger.name":"api.blueprints.drbEdition","thread.id":140159681076096,"thread.name":"MainThread","process.id":2349,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/api/blueprints/drbEdition.py","line.number":43,"entity.type":"SERVICE","error.class":"builtins:Exception","error.message":"Database error"}
